openshift/metrics/setup_grafana.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+datasource_name=prometheus
+
+protocol=http://
+
+datasource_payload="$( mktemp )"
+cat <<EOF >"${datasource_payload}"
+{
+"name": "${datasource_name}",
+"type": "prometheus",
+"isDefault": true,
+"typeLogoUrl": "",
+"access": "proxy",
+"url": "http://$( oc get route lad-prometheus-route -o jsonpath='{.spec.host}' )",
+"basicAuth": false,
+"withCredentials": false
+}
+EOF
+
+cat ${datasource_payload}
```

Comment:
Not sure if this is intended or you included it for testing, but if this is intended I think an echo/printf statement indicating in the terminal output that serves as a description on what the user is looking at would be helpful. E.g. "The Grafana Instance will be configured with the following datasource definition:"  I would suggest the same for the dashboard json at line 40

Reply:
oops that was for debugging, i'll remove that thanks